### PR TITLE
Karmor Profile

### PR DESCRIPTION
**Pull request by PrimalPimmy**
With profiling, we would finally be able to monitor logs in a proper TUI form. The first feature I plan to implement is counting the number of times a file/folder is being accessed by a container. This is similar to the `summary` feature, but this helps the devs monitor the logs without too much effort.

### CODE REVIEW

**Comment by nyrahul** [comment]
@PrimalPimmy , can you provide few screenshots of the feature? Thanks

**Comment by PrimalPimmy** [comment]
> @PrimalPimmy , can you provide few screenshots of the feature? Thanks

@nyrahul  I'm currently in progress to set up a prototype TUI! First time making a TUI, screenshots will done soon by this week hopefully.

**Comment by PrimalPimmy** [comment]
KarmorProfile

Karmor Profile Demo :rocket:

File: profile/Client/Keys.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package profileclient
```
Comment (nyrahul) [inline profile/Client/Keys.go:1]:
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright 2022 Authors of KubeArmor

// profileclient to handle profiling of kubearmor telemetry events
package profileclient
```

File: profile/Client/profileClient.go [hunk 2]
```diff
@@ -0,0 +1,329 @@
+package profileclient
```
Comment (nyrahul) [inline profile/Client/profileClient.go:1]:
```suggestion
// SPDX-License-Identifier: Apache-2.0
// Copyright 2022 Authors of KubeArmor

package profileclient
```

**Review by nyrahul — CHANGES_REQUESTED** [review]
- [ ] fix all lint issues
- [ ] add SPDX license identifier in all files

**Comment by PrimalPimmy** [comment]
Use arrow keys (or h j k l) to scroll through the table (I have to add an help section for this, I will make a prompt help box for this later)

File: profile/Client/profileClient.go [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+)
+
+const (
+	columnNamespace   = "namespace"
+	columnPodname     = "podname"
+	columnProcessName = "ProcName"
+	columnResource    = "Resource"
+	columnResult      = "Result"
+	columnCount       = "count"
+	columnTimestamp   = "timestamp"
```
Comment (daemon1024) [inline profile/Client/profileClient.go:29]: Are these strings column names? We should either keep them all CamelCase or SmallCase for uniformity.

**Review by daemon1024 — CHANGES_REQUESTED** [review]
Noice Work 👌🏽 Full heckerman vibes xD 
hecker-pepe

Can you please check the Linter Warnings in the `Files Changed` Tab? Most of them are related to adding comments for exported Types. Thanks

File: profile/Client/profileClient.go (hunk 3 above)
Comment (PrimalPimmy) [reply profile/Client/profileClient.go:29]: not exactly, these are the "keys" to which a column name is assigned when making a table in bubble tea. And a key is unique to that table column, hence should be a constant with any string of fixed value. This Key communicates between the columns made, and what values should each key get when making a new row.

But yeah I'll make them CamelCase

**Comment by daemon1024** [comment]
@PrimalPimmy You should see them now, the workflow now finished

for context
image

**Comment by daemon1024** [comment]
Still persist. Might wanna install revive tool locally and see the errors.

Can enable it in vscode using `"go.lintTool": "revive",`

**Comment by PrimalPimmy** [comment]
@daemon1024 okay i did that, seems to be fine now (hopefully)

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**